src/Serialization/GAP.jl

```diff
@@ -282,18 +280,19 @@ install_GAP_deserialization(:IsPcGroup,
             length(relord))
       fam = GAPWrap.ElementsFamily(GAPWrap.FamilyObj(F))
       rws = GAP.Globals.SingleCollector(F, GapObj(relord))::GapObj
-      reli = load_object(s, Vector, d[:power_reli], Int)
-      rels = load_object(s, Vector, d[:power_rels], (Vector{Int}, Int))
-      for k in 1:length(reli)
-        GAP.Globals.SetPower(rws, reli[k],
-          GapObj(GAPWrap.ObjByExtRep(fam, GapObj(rels[k]))))
+      if haskey(d, :power_rels)
+        rels = load_typed_object(s, d[:power_rels])
```

Comment:
```suggestion
        loaded_rels = Tuple{Int, Vector{Int}[]
for data_rel in rels
  loaded_rel = load_object(s, Tuple{Int, Vector{Int}, data_rel)
  push!(loaded_rels, loaded_rel)
end
```

Reply:
Ah, I see:
If the data file does not specify the type of the array entries then I can run over the entries and set the type for each of them, instead of trying to specify the type of the whole array.
I will try this.

Reply:
The message for me is:
- Avoid nested vectors.
- Vectors of tuples are fine (although they look like nested vectors in the data files), but deserialize each tuple separately. This way, it is not necessary to store type information about these objects in the data files.